tests/lib/mongo/fixtures/bucketentries.fixtures.ts

```diff
@@ -0,0 +1,72 @@
+import { BucketEntry } from '../../../../lib/core/bucketEntries/BucketEntry';
+import { ObjectId } from 'mongodb';
+import { frames } from './frames.fixtures';
+import { buckets } from './buckets.fixtures';
+
+type MongoBucketEntriesModel = Required<
+  Omit<BucketEntry, 'id' | 'bucket' | 'frame' | 'filename'>
+> & {
+  _id: ObjectId;
+  bucket: ObjectId;
+  name: string;
+  frame?: ObjectId;
+};
+
+const formatBuckeEntries = ({
+  _id,
+  name,
+  frame,
+  ...model
+}: MongoBucketEntriesModel): BucketEntry => {
+  const be: BucketEntry = {
+    ...model,
+    id: _id.toString(),
+    bucket: model.bucket.toString(),
+    filename: name,
```

Comment:
Here its important to note that we are not using `name` but `filename` since storj-service-storage-models changes it to filename (and so the test wouldn't pass without it). However do check in https://github.com/internxt/bridge/pull/115 the filename attribute added to BucketEntry type (not sure if this is what we want) @sg-gs